main.js

```diff
@@ -340,3 +340,37 @@ function paint() {
         ctx.restore();
     }
 }
+
+// copied from utils as unable to import js module in common js
```

Comment:
Is there really no better way of doing this? Could it be imported directly from the HTML? There are additional utility functions in utils.js now like `randSeed` and `mutateBits`...